sshd-core/src/test/java/org/apache/sshd/common/util/io/IoUtilsTest.java

```diff
@@ -54,4 +57,12 @@ public void testGetEOLBytes() {
             assertArrayEquals("Mismatched bytes at iteration " + index, expected, actual);
         }
     }
+
+    @Test
+    public void testGetCurrentUser() throws IOException {
+        // system may be unsupportable, hence the null
+        Collection expected = Arrays.asList(null, System.getProperty("user.name"));
+        String user = IoUtils.getCurrentUser();
+        assertTrue(String.format("Incorrect user got from file owner, expected '%s' got %s", expected, user), expected.contains(user));
+    }
```

Comment:
No need if we agree that _IoUtils.getCurrentUser()_ returns _System.getProperty("user.name")_

BTW what do you mean by 'system may be unsupportable'